softwareComponents/configuration/rofibot.h

```diff
@@ -344,6 +365,7 @@ class NoColision {
  * shoes
  */
 class SimpleColision {
+public:
```

Comment:
Zde chybí implementace, ne?

Reply:
Mea culpa. Bylo nutné rozšířit argumenty pro `operator()` a modulu jsem přidal novou metodu `getOccupiedPositions`.